src/GafferImage/OpenImageIOReader.cpp

```diff
@@ -620,15 +628,25 @@ void OpenImageIOReader::hashChannelData( const GafferImage::ImagePlug *output, c
 	ImageNode::hashChannelData( output, context, h );
 	h.append( context->get<V2i>( ImagePlug::tileOriginContextName ) );
 	h.append( context->get<std::string>( ImagePlug::channelNameContextName ) );
-	hashFileName( context, h );
-	refreshCountPlug()->hash( h );
-	missingFrameModePlug()->hash( h );
+
+	{
```

Comment:
This is what I was expecting, based on the commit message - here we _do_ need context management, because we know the caller has provided a per-tile context, and we want to evaluate some global values.